manual.md

```diff
@@ -12,8 +12,8 @@
 + [1.2. Faster compilation](#section1_2)
 + [1.3. Example usage](#section1_3)
 + [1.4. Dependencies](#section1_4)
-+ [2. Swizzle operators](#section2)
-+ [2.1. Standard C++98 implementation](#section2_1)
++ [2. Swizzling](#section2)
++ [2.1. Default C++98 implementation](#section2_1)
```

Comment:
This said: Not sure anymore! The implementation based on anonymous struct unions used to not be standard but with C++11 this may have changed. 

I guess default is better for now as that would take some time to actually figure out what C++11 supports, what compilers support and when they supported what the anonymous struct unions implementation requires.